client/repo.go

```diff
@@ -4,26 +4,15 @@ package client
 
 import (
 	"fmt"
-	"net/http"
 
 	"github.com/docker/notary"
 	"github.com/docker/notary/trustmanager"
-	"github.com/docker/notary/trustpinning"
 )
 
-// NewNotaryRepository is a helper method that returns a new notary repository.
-// It takes the base directory under where all the trust files will be stored
-// (This is normally defaults to "~/.notary" or "~/.docker/trust" when enabling
-// docker content trust).
-func NewNotaryRepository(baseDir, gun, baseURL string, rt http.RoundTripper,
-	retriever notary.PassRetriever, trustPinning trustpinning.TrustPinConfig) (
-	*NotaryRepository, error) {
-
+func getKeyStores(baseDir string, retriever notary.PassRetriever) ([]trustmanager.KeyStore, error) {
```

Comment:
Non-blocking:  It might be nice to make these exportable; the CLI has a function that basically replicates this for listing keys, etc, and it would be awesome to replace it with these.  Can be done in a separate PR though.